resources/js/ui/InputBar.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env browser */
```

Comment:
Dieser Prototyp ist eventuell unnötig da er nur in dem index  Prototyp aufgerufen wird